app/views/layouts/_head.html.erb

```diff
@@ -20,6 +20,9 @@
   <%= favicon_link_tag "favicon-16x16.png", :rel => "icon", :sizes => "16x16", :type => "image/png" %>
   <%= tag("link", { :rel => "mask-icon", :href => asset_path("tag-icon.svg"), :color => "#7ebc6f" }) %>
   <%= tag("link", { :rel => "manifest", :href => asset_path("manifest.json") }) %>
+  <%= tag("meta", { :name => "mobile-web-app-capable", :content => "yes" }) %> 
```

Comment:
This is [not used anymore](https://developer.chrome.com/multidevice/android/installtohomescreen) in Chrome 39.